### DISCUSSION TITLE
Elevenlabs vocal model update

### DISCUSSION THREAD

**Discussion by kemp-blake** [karashiiro/TextToTalk#195] [Ideas]
The elevenlabs voices are using the "eleven_monolingual_v1" language model during the api request. This is a bit unstable and leaves the voices sounding off (especially if you don't have an american accent).

Would it be possible to change this to "eleven_multilingual_v2" for the better language model?

**Comment by Roctose** [comment]
I'd like to second this. It's level of quality is leagues beyond what the elevenlabs can do, even without accents on the table.

**Comment by Konraddo** [comment]
Nvm, the voices actually appear after game restart.
